Stabilize sparse-registry (#11224)
rust-lang/cargo: src/doc/src/reference/unstable.md

}
```

> ### sparse-registry

Reviewer: Sorry for being picky. For consistency, I guess we can just refer to the new documentation without repeat it. You can take "Namespaced features" as a reference.

```markdown
### Namespaced features

Namespaced features has been stabilized in the 1.60 release.
See the [Features chapter](features.md#optional-dependencies) for more information.
```

[workspace.dependencies](workspaces.md#the-dependencies-table),
and [inheriting-a-dependency-from-a-workspace](specifying-dependencies.md#inheriting-a-dependency-from-a-workspace)
for more information.

### sparse-registry

> Sparse registry support has been stabilized in the 1.66 release.
> 
> The `sparse-registry` feature allows cargo to interact with remote registries served
> over plain HTTP rather than git. These registries can be identified by urls starting with
> `sparse+http://` or `sparse+https://`.
> 
> When fetching index metadata over HTTP, Cargo only downloads the metadata for relevant
> crates, which can save significant time and bandwidth.
> 
> The format of the sparse index is identical to a checkout of a git-based index.

Reviewer 2: I don't think the summary needs to be kept here. Similar to the other sections, it just needs a once sentence explanation, along with a link to the current documentation.

```suggestion
Sparse registry support has been stabilized in the 1.66 release.
See [Sparse Registries](registries.md#sparse-registries) for more information.
```
